src/components/impact-card/ImpactCard.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, Flex, Heading, Text } from "@radix-ui/themes";
+import React from "react";
+
+interface ImpactCardProps {
+  icon: React.ReactNode;
+  value: string;
+  label: string;
+  className?: string;
+}
+
+const ImpactCard: React.FC<ImpactCardProps> = ({
+  icon,
+  value,
+  label,
+  className,
+}) => {
+  return (
+    <Box
+      width="100%"
+      height="248px"
```

Comment:
using `height="248px"` is fragile (localization, dynamic values).

Idea: Prefer e.g. `min-h-[248px]` and let content grow.